e2e/testsuite/grpc_query.go

```diff
@@ -178,7 +177,7 @@ func (s *E2ETestSuite) QueryProposalV1(ctx context.Context, chain ibc.Chain, pro
 
 // GetBlockByHeight fetches the block at a given height. Note: we are explicitly using the res.Block type which has been
 // deprecated instead of res.SdkBlock to support backwards compatibility tests.
```

Comment:
I'm just noticing this comment.

Reply:
ahhh, I knew there was a gotcha, can't believe I missed it :face_exhaling: